Models/Booking/Response/BookingInformationResponse.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+
+namespace StockportGovUK.NetStandard.Models.Booking.Response
+{
+    public class BookingInformationResponse
+    {
+        public Guid Id { get; set; }
+        public string AppointmentName { get; set; }
+        public bool CanCancelCustomer { get; set; }
+        public bool CanCancelAdmin { get; set; }
```

Comment:
Possible name change:
CanCancelCustomer => CanCustomerCancel
CanCancelAdmin => CanAdminCancel

This should be a clearer representation of what these properties are for. Happy to have a discussion about it :)

Reply:
yep this is much clearer will change.